helpers/kernel_config.go

```diff
@@ -283,8 +307,17 @@ func (k *KernelConfig) readConfigFromScanner(reader io.Reader) {
 }
 
 // GetValue will return a KernelConfigOptionValue for a given KernelConfigOption when this is a BUILTIN or a MODULE
-func (k *KernelConfig) GetValue(option KernelConfigOption) (KernelConfigOptionValue, error) {
-	value, ok := k.configs[option].(KernelConfigOptionValue)
+func (k *KernelConfig) GetValue(option interface{}) (KernelConfigOptionValue, error) {
```

Comment:
Why does this take an option as interface{} ?

Reply:
because I have to feed a uint32 from tracee as Im not adding KernelConfigOption (check consts.go in tracee with newly added uint32 const).

Reply:
Okay, it does not have to.. the const can be declared as helper.KernelConfigOption as well I suppose.